ipaplatform/redhat/tasks.py

```diff
@@ -518,9 +514,9 @@ def _create_tmpfiles_dir(self, name, mode, uid, gid):
         os.chmod(name, mode)
         os.chown(name, uid, gid)
 
-    def create_tmpfiles_dirs(self):
+    def create_tmpfiles_dirs(self, ipaapi_user):
```

Comment:
You may copy the config file one time more than necessary. I don't see any harm in that.